modules/signs/src/main/java/eu/cloudnetservice/modules/signs/Sign.java

```diff
@@ -34,7 +30,7 @@
  */
 @ToString
 @EqualsAndHashCode(onlyExplicitlyIncluded = true)
-public class Sign implements Comparable<Sign> {
+public class Sign {
 
   public static final Type COLLECTION_TYPE = new TypeToken<Collection<Sign>>() {
```

Comment:
Can you change this to better way ?

Reply:
Not without breaking compatibility with older spigot versions

Reply:
Can be make a issue.